recipes/xproperty/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DBUILD_TESTS=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
```

Comment:
Same as above. (BTW, it is usually more readable to break that command into multiple lines.)